modular_citadel/code/modules/client/loadout/__donator.dm

```diff
@@ -353,3 +356,128 @@
 
 /obj/item/storage/box/large/custom_kit/davusi/PopulateContents()
 	new /obj/item/card/id/dogtag/donator_davusi(src)
+
+/datum/gear/donator/kits/onelessncrrep
+	name = "My memoir"
+	path = /obj/item/storage/box/large/custom_kit/onelessncrrep
+	ckeywhitelist = list ("hanaharetsu")
+
+/obj/item/storage/box/large/custom_kit/onelessncrrep/PopulateContents()
+	new /obj/item/clothing/head/f13/ncr(src)
+	new /obj/item/card/id/dogtag/onelessncrrep(src)
+
+/obj/item/card/id/dogtag/onelessncrrep
+	name = "Edda's tags"
+	desc = "A dog tag proving enlistment."
+	icon_state = "ncrdogtagrecruit"
+
+/datum/gear/donator/kits/protogen_first_recon
+	name = "1st Recon Screen"
+	path = /obj/item/storage/box/large/custom_kit/protogen_first_recon
+	ckeywhitelist = list ("googlyfox")
+
+/obj/item/storage/box/large/custom_kit/protogen_first_recon/PopulateContents()
+	new /obj/item/clothing/head/beret/donator_worn_beret(src)
+	new /obj/item/clothing/shoes/f13/military/gelatoboots(src)
+	new /obj/item/clothing/mask/gelatoscreen(src)
+	new /obj/item/clothing/under/gelatosuit(src)
+	new /obj/item/gun/energy/laser/wattz(src)
+	new /obj/item/stock_parts/cell/ammo/ec(src)
+	new /obj/item/clothing/suit/armor/bulletproof/gelatocoat(src)
+
+/datum/gear/donator/kits/arbufy
+	name = "A temporary escape"
+	path = /obj/item/storage/box/large/custom_kit/arbufy
+	ckeywhitelist = list ("arbufy")
+
+/obj/item/storage/box/large/custom_kit/arbufy/PopulateContents()
+	new /obj/item/reagent_containers/syringe/stimulants(src)
+	new /obj/item/gun/ballistic/revolver/colt357(src)
+
+/datum/gear/donator/kits/russiansurvivor
+	name = "Our escape from this hell"
+	path = /obj/item/storage/box/custom_kit/russiansurvivor
+	ckeywhitelist = list ("mishanok","skateboreding","vasyano","coolboy20") //drink?
```

Comment:
```suggestion
	ckeywhitelist = list("mishanok","skateboreding","vasyano","coolboy20") //drink?
```
Also, why are four people getting the same loadout...? And a pretty good loadout resources-wise no less.

Reply:
talked with the peeps and restricted it to only one person now